### ISSUE TITLE
Intellij doesn't understand generated test parser

### ISSUE THREAD

**Issue by matozoid** [Strumenta/kolasu#31]
When loading the project all fresh, intellij does not set the source path to the generated test parser. It is in a rather peculiar location if you ask me...

**Comment by ftomassetti** [comment]
Recent versions of IDEA are causing me problems when importing projects. Not sure how to solve this one

**Comment by matozoid** [comment]
Can't we move the generated source to some kind of standard generated source directory?

**Comment by ftomassetti** [comment]
Yes, if there was one. What I did was using the IDEA target to specify that the generated dir should have been added, but that does not work well anymore

**Comment by alessiostalla** [comment]
Gradle seems to be putting some stuff in `build/generated`.
Maven IIRC uses `target/src-generated` or something like that.
